From 37f9f2ab0256356510ab89a6a1a2d34d7ac2c988 Mon Sep 17 00:00:00 2001
From: David Henningsson <david.henningsson@canonical.com>
Date: Thu, 5 Mar 2015 14:55:52 +0100
Subject: [PATCH 2/3] protocol-native: Re-enable srbchannel

Signed-off-by: David Henningsson <david.henningsson@canonical.com>
---
 src/pulsecore/protocol-native.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/pulsecore/protocol-native.c b/src/pulsecore/protocol-native.c
index f54f2a4..e8d147a 100644
--- a/src/pulsecore/protocol-native.c
+++ b/src/pulsecore/protocol-native.c
@@ -5325,7 +5325,7 @@ int pa_native_options_parse(pa_native_options *o, pa_core *c, pa_modargs *ma) {
     pa_assert(PA_REFCNT_VALUE(o) >= 1);
     pa_assert(ma);
 
-    o->srbchannel = false;
+    o->srbchannel = true;
     if (pa_modargs_get_value_boolean(ma, "srbchannel", &o->srbchannel) < 0) {
         pa_log("srbchannel= expects a boolean argument.");
         return -1;
-- 
1.9.1

